Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    from msticpy.nbtools import nbinit
</pre><p>Instead of importing init_notebook you can now do</p><p>import msticpy</p><p>msticp.init_notebook(...</p><p> </p><p>I think this is a bit neater.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>